apps/site/assets/css/_schedule.scss

```diff
@@ -1033,7 +1033,8 @@ del .trip-list-realtime {
       }
     }
 
-    .info-tab {
+    .info-tab,
+    .schedule-tab {
```

Comment:
There is still a schedule tab showing on sm down so I think there is something left to adjust. I think it would be better to do it in the view functions instead of hiding via css. See my other comment on the test.